### PR TITLE
Chat Window Status Bar User Info - #AP-109

### PR DESCRIPTION
**Pull request by davidchan1219**
**What this PR does** 📖

**Which issue(s) this PR fixes** 🔨
I added :server field on the <TailoredCoreStatusbar> component
Fixes #AP-109
Chat Window Status Bar User Info
**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

**Review by Jekrimo — APPROVED** [review]

File: store/friends/actions.ts [hunk 1]
```diff
@@ -152,10 +152,9 @@ export default {
         'removeOutgoingRequest',
         friendAccountToOutgoingRequest(friendAccount).requestId
       )
-      return
+    } else {
+      commit('updateFriend', friend);
```
Comment (stavares843) [inline store/friends/actions.ts:156]: remove ;

not supposed to be used on the project

**Review by stavares843 — CHANGES_REQUESTED** [review]

**Review by stavares843 — COMMENTED** [review]
status bar dynamic content is working ✅ 
Captura de ecrã 2021-11-27, às 22 37 48

File: store/friends/actions.ts [hunk 2]
```diff
@@ -152,10 +152,9 @@ export default {
         'removeOutgoingRequest',
         friendAccountToOutgoingRequest(friendAccount).requestId
       )
-      return
+    } else {
```
Comment (WanderingHogan) [inline store/friends/actions.ts:155]: can you also get rid of this else statement - something like
```
if() {
  // logic
  return
} 
commit('updateFriend', friend)
```

**Review by WanderingHogan — CHANGES_REQUESTED** [review]
use guard pattern instead of else

**Comment by WanderingHogan** [comment]
The last change are the semi colons, everything else looks good!

**Review by RetroPronghorn — APPROVED** [review]

**Review by WanderingHogan — APPROVED** [review]

**Review by stavares843 — APPROVED** [review]

**Pull request merged**